telegram-service/src/app.constants.ts

```diff
@@ -0,0 +1,16 @@
+export const enum SceneNames {
+    INPUT_CONTRACT_SCENE = 'INPUT_CONTRACT_SCENE'
+  };
+
+  export const enum Actions {
+    BID = 'bid',
```

Comment:
in one place you use capsed naming in another lowercase
choose one, and i suggest uppercase